### PR TITLE
Remove specialist document types

### PR DESCRIPTION
**Pull request by steventux**
Please don't merge yet, we'd like to run this branch on integration for some end-to-end testing.

https://trello.com/c/B9cw4NZY/292-clean-up-finders-code-in-v1-medium
Less hexagons.
Remove any redundant code now this application is used for the sole purpose of publishing manuals.

### CODE REVIEW

**Comment by kevindew** [comment]
@steventux When you merge/rebase against master (assuming this is merged after https://github.com/alphagov/manuals-publisher/issues/757) could you remove references usage of `GdsApi::TestHelpers::PublishingApi` and it's methods `stub_default_publishing_api_*` in [publishing_api_helpers](https://github.com/alphagov/manuals-publisher/blob/master/features/support/publishing_api_helpers.rb)? these won't be needed once your code is removed.

**Comment by steventux** [comment]
@kevindew there's a test dependency on `GdsApi::TestHelpers::PublishingApi` as a couple of methods are used from this lib. I have removed the stubbing for v1 stuff though.

**Comment by kevindew** [comment]
@steventux Thanks - sure, we'll have to have a look at removing those when we come to decommission v1 endpoints.

File: README.md [hunk 1]
```diff
@@ -25,13 +25,22 @@ Publishing App for Manuals.
...
-* [alphagov/email-alert-api](http://github.com/alphagov/email-alert-api): sends emails to subscribed users when documents are published
 
 ## Running the application
 
+To run the application in development you will need at least one user in the application database. 
+In a rails console do:
+
+```
+User.create!(name: "My Name", email: "my.email@somedomain.com", permissions: ["gds_editor"], organisation_slug: "government-digital-service", organisation_content_id: "af07d5a5-df63-4ddc-9383-6a666845ebe9")
+```
```
Comment (kevindew) [inline README.md]: 👍

**Pull request merged**